tests/react/client/components/drawers/index.test.tsx

```
@@ -0,0 +1,71 @@
+    setup('settings')
+
+    const mockSettings = screen.getByTestId('mock-settings')
+    fireEvent.click(mockSettings)
+
+    // Test passes if no error is thrown
+    expect(true).toBe(true)
+  })
+})
+
+// Named export to comply with codebase rules
+export { }
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)